docs/who_are_we.md

```diff
@@ -0,0 +1,39 @@
+
+# Who Are We ?
+
+## Foreword
+
+We are excited to count among our ranks numerous key contributors to so many key project of the Python Open Source Ecosystem.
+So many companies / groups / organizations who recognized the value of fixing the issues WheelNext is trying to approach.
+
+Below you will find the list of communities, projects and companies involved in making WheelNext great !
+
+## Open-Source Projects (alphabetic order)
+
+- AstroPy: https://www.astropy.org/
+- Conda: <https://docs.conda.io/projects/conda/en/stable/>
+- GPU-Mode: <https://github.com/gpu-mode>
```

Comment:
```suggestion
- GPU MODE: <https://github.com/gpu-mode>
```

Reply:
(Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)